telos-frontend/src/components/steam/steam.jsx

```diff
@@ -42,19 +49,46 @@ const Steam = () => {
       getSteamData(steamName);
     }
   }, []);
+  const NoDataColumn = [
+    { field: 'StatusNow', headerName: `Status: ${dataMessage}`, width: 600, sortable: false },
+  ];
+
+  const NoDataRows = [
+    {
+      id: 'Requirements1',
+      StatusNow: 'Looks like somthing went wrong. Try the following:',
+      sortable: false,
+    },
+    {
+      id: 'Requirements2',
+      StatusNow: '1. Make sure the steam profile you are searching is public',
+      sortable: false,
+    },
+    { id: 'Requirements3', StatusNow: '2. Create a bug report here: ', sortable: false },
+    {
+      id: 'BugReport',
+      StatusNow:
+        'https://github.com/UOA-SE701-Group3-2021/3Lancers/issues/new?assignees=&labels=&template=bug_report.md&title=',
```

Comment:
Would it be possible to make this and line 81 into links? Just so that they wouldn't have to copy and paste the URLs

Reply:
I think I tried to make it as a link but did not have much luck.